src/app/home/featured/featured.component.pug

```diff
@@ -1,8 +1,8 @@
-.row#featured-container.mx-0((mouseenter)="pauseSlideShow()", (mouseleave)="runSlideshow(primaryFeaturedIndex)")
+.row#featured-container.mx-0
   //- Primary Image
   .col-sm-9.featured-col#featured-primary
     ng-container(*ngFor="let i of [0,1,2]")
-      a#featuredCollectionLink.card.featured-card([class.fade-in]="primaryFeaturedIndex == i", [attr.href]="featured[i].link | translate", tabindex="-1")
+      a.card.featured-card.hidden-text([class.fade-in]="primaryFeaturedIndex == i", [attr.href]="featured[i].link | translate", tabindex="-1" data-qa="featuredCollectionLink" style="font-sixe:0") {{ featured[i].link_title | translate }}
```

Comment:
font-sixe? 🙈

Reply:
oh yeah, I need to get rid of that because of the added class. Thanks!